src/HotelAdministrator.java

```diff
@@ -0,0 +1,44 @@
+package Task_4;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HotelAdministrator {
+    private List<Room> rooms;
+    private List<Service> services;
+
+    public HotelAdministrator() {
+        this.rooms = new ArrayList<>();
+        this.services = new ArrayList<>();
+    }
+
+    public void checkIn(Room room) {
+        room.setStatus(RoomStatus.OCCUPIED);
```

Comment:
Look here you changing room status only for room you passing to method, it is not changing in List<Room>. So after you execute this method and try to look for all rooms statuses they will not be changed. 
In this method, you should receive Room from RoomsList and work with it. So after setting status to this room it actually affects HotelAdministator object, not only room object